src/Console/ModelsCommand.php

```diff
@@ -467,8 +467,9 @@ public function getPropertiesFromTable($model)
             }
             $databasePlatform->registerDoctrineTypeMapping($yourTypeName, $doctrineTypeName);
         }
-
-        $database = null;
+        
+        $connectionName = $model->getConnection()->getName();
+        $database = config("ide-helper.db_mapping.$connectionName");
```

Comment:
I have not checked the code yet, but it's wrong to use the global helper here; see all the other instances in this file how they access the config (resolve via container):

```php
$this->laravel['config']->get();
```
This is important as not everyone runs this library on a full framework installation, some might only use the illuminate/ components and not have this defined.

Reply:
Correct, sorry about that. This is now fixed.